docs/sources/flow/reference/components/discovery.gce.md

```diff
@@ -0,0 +1,78 @@
+---
+title: discovery.gce
+---
+
+# discovery.gce
+
+`discovery.gce` allows retrieving scrape targets from [GCP GCE](https://cloud.google.com/compute) instances. The private IP address is used by default, but may be changed to the public IP address with relabeling.
```

Comment:
Spell out acronyms the first time they're used (from Google's documentation style guide on [Abbreviations](https://developers.google.com/style/abbreviations#spelling-out)):

```suggestion
`discovery.gce` allows retrieving scrape targets from [Google Compute Engine](https://cloud.google.com/compute) (GCE) instances. The private IP address is used by default, but may be changed to the public IP address with relabeling.
```